src/languages/swift.js

```diff
@@ -283,6 +283,49 @@ export default function(hljs) {
   };
   TYPE.contains.push(GENERIC_ARGUMENTS);
 
+  // https://docs.swift.org/swift-book/ReferenceManual/Declarations.html#ID380
+  const OPERATOR_DECL = {
+    beginKeywords: 'operator',
+    contains: [
+      {
+        match: /\s+/,
+        relevance: 0
+      },
+      {
+        className: 'title',
+        match: Swift.operator,
+        endsParent: true,
+        relevance: 0
+      }
+    ]
+  };
+
+  // https://docs.swift.org/swift-book/ReferenceManual/Declarations.html#ID550
+  const PRECEDENCEGROUP = {
+    beginKeywords: 'precedencegroup',
+    contains: [
+      {
+        match: /\s+/,
+        relevance: 0
+      },
+      {
+        className: 'title',
+        match: Swift.typeIdentifier,
+        relevance: 0
+      },
+      {
+        begin: /{/,
+        end: /}/,
+        endsParent: true,
+        keywords: [
+          ...Swift.precedencegroupKeywords,
+          ...Swift.literals
+        ].join(' '),
+        contains: [ TYPE ]
+      }
```

Comment:
```suggestion
      {
        begin: /{/,
        end: /}/,
        relevance: 0,
        endsParent: true,
        keywords: [
          ...Swift.precedencegroupKeywords,
          ...Swift.literals
        ].join(' '),
        contains: [ TYPE ]
      }
```

The match group isn't what's relevant, the keywords inside are.

Reply:
Fixed.